### DISCUSSION TITLE
One node has "Cannot find disk config file, maybe due to a mount error"

### DISCUSSION THREAD

**Discussion by lanwin** [longhorn/longhorn#2610] [Q&A] [answered]
Hi, longhorn works well here except that I have one node witch is not schedulabe and can not get the reason why.

The condition is not ready with the following message:
> Last Transition Time: 6 months ago
> Message: Disk default-disk-81000000000(/var/lib/longhorn/) on node kubenode04 is not ready: cannot find disk config file, maybe due to a mount error
> Reason: DiskFilesystemChanged
> Status: False

The device on that node is successfully mounted and is writeable.

What can I do about this?

**Comment by cclhsu** [comment]
HI!  Does the volume on the was evicted and add back before?

**Reply by lanwin** [reply]
No but it could be that the device was not mounted to the node at the time I started longhorn.

**Comment by lanwin** [comment, marked as answer]
I found the solution myself. What I didnt noticed was that I could delete the default volume (tiny trash icon). After disable scheduling I could delete the volume and create a new one with the same location and everything works like expected.
